cores/esp32/esp32-hal-log.h

```diff
@@ -125,9 +130,24 @@ int log_printf(const char *fmt, ...);
 #define log_n(format, ...)
 #define isr_log_n(format, ...)
 #endif
+#endif
 
 #include "esp_log.h"
 
+#ifdef USE_ESP32_LOG
+
+#ifndef TAG
+#define TAG "ESP32"
```

Comment:
using TAG like "arduino" makes more sense here

Reply:
No problem , i will change it for sure :)